### ISSUE TITLE
Is the image 32 bit or 64 bit?

### ISSUE THREAD

**Issue by mianasbat** [pypa/manylinux#557]
Hi I was exploring the 32bit image `quay.io/pypa/manylinux2014_i686` but it shows 64 bit architecture on running. Could you please let me know if it is a 32 bit or 64 bit. Or is it like 64 bit working on 32 bit capacity.      

The steps I took to find the architecture are      
```
docker pull quay.io/pypa/manylinux2014_i686
docker run -it quay.io/pypa/manylinux2014_i686 bash
uname -m
```

The other image I tried `quay.io/pypa/manylinux1_i686` gives 32 bit architecture but it is Centos 5 and too old for me to work on. The above one is Centos 7.7.

**Comment by mayeut** [comment]
The image only contains 32bit software but was missing the `linux32` entry point thus reporting as `x86_64`

**Comment by mianasbat** [comment]
I am not sure what that means, I was trying to install gcc > 5, the default one is 4.8. However, I couldnt install and then I found that 32 and 64 bit thing. Any idea?

**Comment by mayeut** [comment]
for gcc > 5, you'll have better luck with manylinux2010.
centos now has little support for i686, including the devtoolset that's not supported (used on other arch that provide gcc 8).

**Comment by mayeut** [comment]
closed by #563
